SpatialGDK/Source/SpatialGDKFunctionalTests/Private/SpatialFunctionalTest.cpp

```diff
@@ -783,6 +783,7 @@ void ASpatialFunctionalTest::DeleteActorsRegisteredForAutoDestroy()
 APlayerController* ASpatialFunctionalTest::GetFlowPlayerController()
 {
 	ASpatialFunctionalTestFlowController* FlowController = GetLocalFlowController();
+	checkf(IsValid(FlowController), TEXT("FlowController must be valid. You may be calling this on a server."));
 	if (!IsValid(FlowController))
```

Comment:
I've left the IsValid conditional in as I know checkf doesn't run in shipping builds, is that needed or can I remove it?

Reply:
So, had to go here to check https://docs.unrealengine.com/en-US/ProgrammingAndScripting/ProgrammingWithCPP/Assertions/index.html
but I think we can do ensureAlwaysMsgf here as the condition

Reply:
you could collapse this to `if(!ensureAlways...)` but may make it less readable, leaving it up to you

Reply:
o that also returns bool - done now